src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -24139,15 +24139,15 @@ interface I { }
         }
 
         [Fact]
-        public void RecordLoadedInVisualBasicDisplaysAsRecord()
+        public void RecordLoadedInVisualBasicDisplaysAsClass()
         {
             var src = @"
 public record A;
 ";
             var compRef = CreateCompilation(src).EmitToImageReference();
             var vbComp = CreateVisualBasicCompilation("", referencedAssemblies: new[] { compRef });
             var symbol = vbComp.GlobalNamespace.GetTypeMember("A");
```

Comment:
Could you add an assert for `IsRecord` here as well.

Reply:
Updated, thanks for the review!